editor/project_manager.cpp

```diff
@@ -2656,6 +2661,16 @@ ProjectManager::ProjectManager() {
 		erase_ask->get_ok_button()->connect("pressed", callable_mp(this, &ProjectManager::_erase_project_confirm));
 		add_child(erase_ask);
 
+		VBoxContainer *erase_ask_vb = memnew(VBoxContainer);
+		erase_ask->add_child(erase_ask_vb);
+
+		erase_ask_label = memnew(Label);
+		erase_ask_vb->add_child(erase_ask_label);
+
+		delete_project_contents = memnew(CheckBox);
+		delete_project_contents->set_text(TTR("Also delete project contents"));
```

Comment:
The message should be more dramatic since there's a risk of data loss :wink: 

```suggestion
		delete_project_contents->set_text(TTR("Also delete project contents from the filesystem (no undo!)"));
```

Reply:
Should it say no undo even though it moves them to the systems trash (although I know not all people have a trash/recycle bin)

Reply:
Alright, I added "(no undo!)" to both pr's